packages/traceloop-sdk/traceloop/sdk/evaluator/stream_client.py

```diff
@@ -0,0 +1,91 @@
+import httpx
+import json
+import os
+
+from .model import ExecutionResponse
+
+
+class SSEClient:
+    """Handles Server-Sent Events streaming"""
+
+    def __init__(self, shared_client: httpx.AsyncClient):
+        self._api_endpoint = os.environ.get(
```

Comment:
do not get it here, you should get it from the outside - users may set the URL and api key when initializing traceloop